src/modules/auth/controllers/het-archief.controller.ts

```diff
@@ -108,6 +114,12 @@ export class HetArchiefController {
 				statusCode: HttpStatus.TEMPORARY_REDIRECT,
 			};
 		} catch (err) {
+			if (err?.message === 'SAML Response is no longer valid') {
```

Comment:
Waarom zou err hier niet bestaan?

Reply:
Ik voeg die altijd toe als ik niet zeker ben van het object dat ik zal krijgen.
Normaal zal die wel altijd bestaan, zelfs als het een leeg object is of een string.

Als je een vraagteken teveel zet, dan crashed de app niet
Als je een vraagteken te weinig zet, dan crashed de app

dus gewoon als voorzorg

Reply:
Wel ik probeer voor de testen van de meeste files (controllers, services, helpers...) aan 100% coverage te geraken, dus per vraagteken is dat een test extra om die branch ook te hitten. dus als je een test bij schrijft, geen probleem voor mij.
Maar daarom laat ik die ook rapper weg als het een situatie is die zich eigenlijk niet zal voordoen.